pkg/types/types.go

```diff
@@ -86,15 +119,15 @@ func numZeros(ip []byte) uint8 {
 }
 
 // IPStringToBytes creates a goDB compatible bytes slice from an IP address string
-func IPStringToBytes(ip string) ([]byte, error) {
+func IPStringToBytes(ip string) ([]byte, bool, error) {
```

Comment:
Nitpick: this function could benefit from either named return parameters or a note in the description that we return whether the IP string is an IPv4 string.

Reply:
Fair point. Did both and added a few simple unit tests.